api/controller/dictionary.py

```diff
@@ -16,7 +17,10 @@ def _search(self, word, language):
     definitions = self.combine_definitions(es_results)
     return definitions
 
-  def definition(self, word, language="English"):
+  def definition(self, word, language="en"):
+    if len(language) != 2 or len(language) != 3:
+      language = self.update_language_code(language)
```

Comment:
With this code, an alpha-3 won't be converted into alpha-2. If you always want to get alpha-2, I think you should call `update_language_code` something like `normalize_language_code` and call it every time.